advisories/5.4.2/BRSA-i3wlupztu5jm.toml

```diff
@@ -1,7 +1,7 @@
 [advisory]
 id = "BRSA-i3wlupztu5jm"
-title = "nvidia-container-toolkit CVE‑2025‑23359"
-cve = "CVE‑2025-23359"
+title = "nvidia-container-toolkit CVE-2025-23359"
+cve = "CVE-2025-23359"
```

Comment:
Can you open an issue somewhere - Twoliter? - to add a lint for this?

Stuff like this isn't reasonable to expect humans to catch in review, so this will keep happening unless we have automated checks, or else more tolerant downstream tools.

Reply:
Something like:
```
#!/bin/bash

if find advisories -name '*.toml' -type f >/dev/null 2>&1 ; then
  export LC_ALL=C
  if grep --include '*.toml' -R -l -P ['\x80'-'\xFF'] advisories ; then
    echo "error: found non-ASCII characters in advisories" >&2
    exit 1
  fi
fi

❯ make twoliter check-advisories
advisories/1.0.4/BRSA-6jzsr8lqncrq.toml
advisories/1.0.4/BRSA-tdebxkarxymw.toml
advisories/1.0.4/BRSA-4mz3mlhteng6.toml
advisories/1.0.4/BRSA-jd3wf9wpwu5l.toml
advisories/1.0.4/BRSA-bl2apnymyndl.toml
error: found non-ASCII characters in advisories
```

Reply:
Opened https://github.com/bottlerocket-os/twoliter/issues/466 to get some lints into Twoliter

Reply:
Opened PR https://github.com/bottlerocket-os/twoliter/pull/467 to add a lint task for twoliter. Also merged https://github.com/bottlerocket-os/bottlerocket-kernel-kit/pull/55 to fix this same issue in the kernel-kit.